lib/src/connector/score_connector.dart

```diff
@@ -141,13 +135,18 @@ class ScoreConnector {
       result = await Connector.getDataByGet(parameter);
       tagNode = parse(result);
       rankNodes = tagNode.getElementsByTagName("tbody")[0].getElementsByTagName("tr");
-      rankNodes = rankNodes.getRange(2, rankNodes.length).toList().reversed.toList();
+      final List<Element> filteredRankNodes = [];
+      for (final rankNode in rankNodes) {
+        if (rankNode.getElementsByTagName("td").length < 7) continue;
+        filteredRankNodes.add(rankNode);
+      }
+      rankNodes = filteredRankNodes;
+      rankNodes = rankNodes.toList().reversed.toList();
```

Comment:
```suggestion
      rankNodes = filteredRankNodes.reversed.toList();
```

Suggest eliminating unneeded `toList` conversions. The `reverse` method is actually belongs to `Iterable`, which is the base type of `List`.

The `reversed` method defined in `Iterable` like below:

```dart
  /// An [Iterable] of the objects in this list in reverse order.
  /// ```dart
  /// final numbers = <String>['two', 'three', 'four'];
  /// final reverseOrder = numbers.reversed;
  /// print(reverseOrder.toList()); // [four, three, two]
  /// ```
  Iterable<E> get reversed;
```

In addition, we may consider to reduce the number of re-assigning value to a same variable. (It may cause memory re-allocation if the compiler not smart enough...)

Reply:
commit https://github.com/NEO-TAT/tat_flutter/commit/9325e4f3cd8d5c53d4adefc380817dac53e1e7b1

Reply:
ca584f88d7d5c2bb6a9de9a4aed25a3deea0a093